app/Http/Controllers/ResumeController.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+declare(strict_types=1);
+
+// phpcs:disable Generic.CodeAnalysis.UnusedFunctionParameter,SlevomatCodingStandard.Functions.UnusedParameter
+
+namespace App\Http\Controllers;
+
+use App\User;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Facades\Storage;
+use App\Http\Requests\StoreResumeRequest;
+
+class ResumeController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('permission:read-users-resume|read-users-own', ['only' => ['showResume']]);
+        $this->middleware('permission:update-users-resume|update-users-own', ['only' => ['storeResume']]);
+        $this->middleware('permission:delete-users-resume|update-users-own', ['only' => ['deleteResume']]);
+        $this->middleware('permission:read-users-resume', ['only' => ['showResumeBook']]);
+    }
+
+    /**
+     * Show the user's resume.
+     *
+     * @param string $id
+     *
+     * @return \Illuminate\Http\JsonResponse|\Symfony\Component\HttpFoundation\BinaryFileResponse
+     */
+    public function showResume(string $id)
```

Comment:
Once the resume book function has been extracted to another controller, this should be renamed to remove `Resume` from the function name.